### PR TITLE
fix: Outdated template language codes

### LINKED ISSUE #55: SupportedLanguageCodes for templates do not match the meta docs
The language codes for templates do not match the documented supported language codes, some are missing and others are simply not supported according to the docs.

https://developers.facebook.com/docs/whatsapp/business-management-api/message-templates/supported-languages/ 
https://developers.facebook.com/docs/whatsapp/api/messages/message-templates/#supported-languages

### PR DESCRIPTION
**Pull request by lukas-becker0**
Closes #55.

### CODE REVIEW

File: src/types/languageCodes.ts [hunk 1]
```diff
@@ -1,104 +1,78 @@
...
-	WesternFrisian: "fy",
-	Xhosa: "xh",
-	Yiddish: "yi",
-	Yoruba: "yo",
-	Zulu: "zu",
-} as const;
+// See the docs
+// here: https://developers.facebook.com/docs/whatsapp/business-management-api/message-templates/supported-languages/
+// and/or here: https://developers.facebook.com/docs/whatsapp/api/messages/message-templates/#supported-languages
+export declare const SUPPORTED_LANGUAGE_CODES: {
```
Comment (MarcosNicolau) [inline src/types/languageCodes.ts:4]: I don't like this syntax, could you declare it normally (as it was before) and use `as const` instead please.

**Review by MarcosNicolau — APPROVED** [review]

File: src/types/languageCodes.ts (hunk 1 above)
Comment (lukas-becker0) [reply src/types/languageCodes.ts:4]: Ups yes sorry, that was not my intention.

Comment (MarcosNicolau) [reply src/types/languageCodes.ts]: Thanks

**Review by MarcosNicolau — APPROVED** [review]
Thanks

**Pull request merged**